assets/src/models/detoursList.ts

```diff
@@ -1,16 +1,18 @@
 import { array, Infer, nullable, number, string, type } from "superstruct"
 
+export type DetourId = number
```

Comment:
Partially, I would prefer to make the id type opaque to the frontend, but in the case that we ever decide to change the key, I didn't want to track everything down, but also we have an existing practice of making types for ID's so that it's clear when you can compare IDs (because it wouldn't make sense to compare a Detour ID to a Route ID)